front/src/Components/Profile.css

```diff
@@ -2,28 +2,30 @@
   font-family: Arial, sans-serif;
 }
 
-.profile{
+.profile {
   display: flex;
   flex-direction: column;
   align-items: center;
   text-align: center;
   border: solid #aaa 1px;
   border-radius: 5%;
   margin: 11vh 10vw 1vh 10vw;
-  background: #e6e6e6;
+  background: var(--other-background);
 }
 
 .profilePic {
   border-radius: 50%;
   height: 30vw;
   width: 30vw;
   padding: 3%;
-  background: #e6e6e6;
+  background: var();
```

Comment:
`var(--other-background)`?

Reply:
Yes I forgot I just modify in local, I push it immediately